docs/installation/windows.md

```diff
@@ -59,7 +60,11 @@ and virtualization support is enabled in BIOS and recognized by Windows.
 	  target="_blank"> Microsoft® Hardware-Assisted Virtualization Detection
 	  Tool</a> and follow the on-screen instructions.
 
-
+3. Verify your Windows OS is 64-bit (x64)
+   How you do this verification depends on your Windows version.  For details, see the Windows 
```

Comment:
I think there should be a blank line before "How you do this", because that line is now rendered directly after `(x64)`